src/scss/components/checkbox/mixins/_hxCheckboxControl.scss

```diff
@@ -0,0 +1,112 @@
+@import "vars";
+@import "hxCheckboxFacade";
+
+//
+// ===== Mixins for <hx-checkbox-control> states =====
+//
+
+// ========== PRISTINE ==========
+@mixin __checkboxControl($pseudo-state: null) {
+  @if $pseudo-state == focus {
+    box-shadow: $focus-glow;
+
+    &::-ms-check {
+      border-color: $cyan-700;
+    }
+
+    ~ label > hx-checkbox {
+      border-color: $cyan-700;
+      box-shadow: $focus-glow;
+    }
+  }
+  @else if $pseudo-state == hover {
+    color: $cyan-500;
+
+    &::-ms-check {
+      background-color: $cyan-50;
+      color: $cyan-500;
+    }
+  }
+}
+
+// ========== INVALID ==========
+@mixin __checkboxControl--invalid($pseudo-state: null) {
+  color: $red-900;
+
+  &::-ms-check {
+    border-width: 2px;
+    border-color: $red-900 !important;
+    color: $red-900 !important;
+  }
+
+  @if $pseudo-state == focus {
+    box-shadow: $focus-glow-invalid;
+
+    &::-ms-check {
+      border-color: $red-900;
+    }
+
+    ~ label > hx-checkbox {
+      border-color: $red-900;
+      box-shadow: $focus-glow-invalid;
+      border-width: 2px;
+    }
+  }
+  @else if $pseudo-state == hover {
+    &::-ms-check {
+      background-color: $red-100;
+    }
+
+    ~ label > hx-checkbox {
+      border: 2px solid $red-900;
+      color: $red-900;
```

Comment:
What combination of `$state` and `$pseudo-state` will these styles be applied?

I think these apply to the default `invalid` state, meaning that this is probably not the place you would want to define these styles.  Additionally, these styles should be applied via `hxCheckoxFacade(invalid)` in `src/scss/components/checkbox/_index.scss`.